### DISCUSSION TITLE
[Wiki]: Proposal to clarify Dockerless install with asdf version manager

### DISCUSSION THREAD

**Discussion by SethFalco** [wg-easy/wg-easy#1381] [Q&A]
### Description

Proposing this as an issue as the wiki does not allow public edits.

In the page, [Using WireGuard Easy without Docker](https://github.com/wg-easy/wg-easy/wiki/Using-WireGuard-Easy-without-Docker), it clarifies how to run the services but assumes that the package managers version of `node` meets the requirements.

Some users such as myself use [asdf](https://github.com/asdf-vm/asdf), there are others that use [nvm](https://github.com/nvm-sh/nvm), etc. This is because the first-party repos on Ubuntu still serves Node.js v12.

It seems the instructions don't work when Node.js is installed through a version manager, despite being configured properly in the root user's `.bashrc`.

I'm able to run the service manually (as it uses the correct `node` binary), but systemctl/systemd fails to.

### Suggested Solution

Clarify explicitly in the docs that if one has `node` installed via a version manager like asdf or nvm, to specify a path to the binary in the version managers directory instead of `node` command.

For example:

```Desktop
ExecStart=/usr/bin/env /root/.asdf/installs/nodejs/22.7.0/bin/node /app/server.js
```

### Alternatives

Advice welcome if you have a better solution for this!

**Comment by kaaax0815** [comment]
there are multiple reasons to deprecate Dockerless install altogether (or make it more manual)

**Comment by kaaax0815** [comment]
Dockerless deprecation is in progress. This is now a advanced topic. Converting to discussion
